Sources/Emul8080r/CPU.swift

```diff
@@ -31,17 +31,19 @@ public final class CPU {
 
     public weak var bus: IOBus?
 
-    public internal(set) var state = State8080()
-    public internal(set) var memory: [UInt8]
+    public internal(set) var state: State8080
 
-    public init(memory: [UInt8], systemClock: SystemClock) {
-        self.memory = memory
+    let safeMemoryBounds: Range<UInt8>?
```

Comment:
Originally the safeMemoryBounds was hard-coded to the Space Invaders memory layout, this doesn't make sense for the emul8080r in isolation.